includes/class-wc-payments.php

```diff
@@ -852,8 +846,6 @@ public static function maybe_register_platform_checkout_hooks() {
 			add_filter( 'woocommerce_cookie', [ __CLASS__, 'determine_session_cookie_for_platform_checkout' ] );
 			// Disable nonce checks for API calls. TODO This should be changed.
 			add_filter( 'woocommerce_store_api_disable_nonce_check', '__return_true' );
-			add_action( 'woocommerce_checkout_billing', [ __CLASS__, 'platform_checkout_fields_before_billing_details' ], -50 );
-			add_filter( 'woocommerce_form_field_email', [ __CLASS__, 'filter_woocommerce_form_field_platform_checkout_email' ], 20, 4 );
```

Comment:
These 2 action and filter will load after the cart is loaded; after the `woocommerce_cart_loaded_from_session` hook.